src/Discord/Internal/Types/Guild.hs

```diff
@@ -119,26 +130,42 @@ instance FromJSON GuildUnavailable where
        GuildUnavailable <$> o .: "id"
 
 data PresenceInfo = PresenceInfo
-  { presenceUserId     :: UserId
+  { presenceUserId     :: Maybe UserId
```

Comment:
Is the UserID optional now? As in, are we running into crashes because the current code assumes the id is always present? if so merging sooner is better than any nitpicks in this review